src/components/Library/MonthVoting.tsx

```diff
@@ -99,17 +86,12 @@ const StEndedVotingListWrapper = styled.section`
 
   width: 100%;
   margin-top: 1.7rem;
```

Comment:
요렇게 하면 마진 이슈 해결되는거같아!!!
```suggestion
  display: flex;

  width: 100%;
  padding-left: 2rem;
  margin-top: 1.7rem;
```

Reply:
flex,margin은 위에 있어서 padding만 적용해써! 고마어!